pkg/olog/olog.go

```diff
@@ -115,6 +116,11 @@ func NewWithHandler(h slog.Handler) *slog.Logger {
 	return slog.New(h)
 }
 
+// SetOutput sets the global logger output to desired writer.
+func SetOutput(w io.Writer) {
+	defaultOut = w
+}
```

Comment:
This isn't thread-safe. You'll probably want to call this out in the function signature, or have a lock protecting `defaultOut`.